frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -78,6 +78,19 @@ export default function AppNavbar({ currentUser, systemInfo, doLogout, currentUr
               }
             </Nav>
 
+            <Nav className="mr-auto">
+              {
+                hasRole(currentUser, "ROLE_USER") && (
+                  <NavDropdown title="Hotels" id="appnavbar-hotels-dropdown" data-testid="appnavbar-hotels-dropdown">
+                    <NavDropdown.Item href="/hotels/list" data-testid="appnavbar-hotels-list">List</NavDropdown.Item>
```

Comment:
This should link to "/hotels" instead of "/hotels/list"; see App.js

Reply:
Fixed. Thank you.